test/parallel/test-runner-mock-timers.js

```diff
@@ -351,6 +357,65 @@ describe('Mock Timers Test Suite', () => {
         assert.deepStrictEqual(order, ['f1', 'f2']);
       });
     });
+
+    describe('process.nexTick Suite', () => {
+      it('should keep nextTick working if timers are disabled', (t, done) => {
+        const now = Date.now();
+        const timeout = 2;
+        const expected = () => now - timeout;
```

Comment:
I am confused, shouldn't this be `const expected = () => Date.now() - timeout;`?
Currently it asserts the same constant values if I understand correctly